src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -605,6 +627,25 @@ private PrecompileArguments GetPrecompileArgumentsFor(ITaskItem assemblyItem, st
             aotArgs.Add($"static");
         }
 
+        if (UseDirectPInvoke)
+        {
+            aotArgs.Add($"direct-pinvoke");
+        }
+
+        if (DirectPInvokes.Length > 0)
+        {
+            var directPInvokesSB = new StringBuilder("direct-pinvokes=");
+            Array.ForEach(DirectPInvokes, directPInvokeItem => directPInvokesSB.Append($"{directPInvokeItem.GetMetadata("Identity")};"));
```

Comment:
nit: you can use ItemSpec property directly instead of going over GetMetadata for Identity value.